index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+</head>
+<body style=" background-color: #a5d3eb;">
```

Comment:
It's better to let the style to "styles.css"

Reply:
Please consider always giving a reason that `why you are asking for a change`. It could be benefits, best practices, or requirements. You can use the following suggestion here: -

> "Please consider using a separate file for your styles rather than inline styles as its considered to be a bad practice to use inline styles"